src/os/bluestore/BlueStore.cc

```diff
@@ -18662,10 +18658,27 @@ void BlueStore::ExtentMap::provide_shard_info_to_onode(bufferlist v, uint32_t sh
   }
 }
 
+//-----------------------------------------------------------------------------------
+void BlueStore::set_allocation_in_simple_bmap(SimpleBitmap* sbmap, uint64_t offset, uint64_t length)
+{
+  //dout(10) << "offset=" << offset << ", length=" <<length<< ", min_alloc_size=" <<min_alloc_size
+  //	   << ", min_alloc_size_mask=" << min_alloc_size_mask << dendl;
+#if 0
```

Comment:
I changed code which is used elsewhere because I don't the case could ever happen (if if it will the old code will break), but I still want to keep a reference to the previous code

Reply:
It isn't a good idea to leave dead code around - either '#if 0' or commented-out.
If you want to explain a change from a previous version - you can add relevant text in a comment, explaining
what was changed and why.